src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPreferenceType.java

```diff
@@ -9,14 +9,14 @@
 @AllArgsConstructor
 public enum ContactPreferenceType implements HasLabel {
 
-
-
     @JsonProperty("Email")
     EMAIL("Email", "Email"),
     @JsonProperty("Post")
     POST("Post", "Post");
+
     private String type;
     private final String label;
+
     public boolean isEmail() {
         return EMAIL.name().equalsIgnoreCase(this.name());
```

Comment:
This has the same problems as the isPost() method originally proposed - could we fix this as part of this change?
Or else raise a tech debt ticket...